### PR TITLE
[mesa] new versions: bugfix 21.2.6, minor 21.3.1

### PR DESCRIPTION
**Pull request by wdconinc**
This adds the recent bugfix version of the 21.2 series, and the first recommended version of the 21.3 series. Per the mesa release notes, 21.3.0 is not recommended for stable installations, so it is not included in the list of versions in package.py.

As reported in #25913, `mesa` depends on `unwind`, which is made explicit here. The discussion in that PR indicates that `libunwind` works on `aarch64` and `arm`. There are no conflicts in `libunwind` to indicate otherwise. There does not seem to be a way to specify a `depends_on('unwind', when='target!=arm')` so this now effectively introduces the `unwind` dependency for all targets, but (as before) does not use it for `arm` and `aarch64`.

References:
- [21.2.6 release notes](https://gitlab.freedesktop.org/mesa/mesa/-/blob/main/docs/relnotes/21.2.6.rst)
- [21.3.0 release notes](https://gitlab.freedesktop.org/mesa/mesa/-/blob/main/docs/relnotes/21.3.0.rst) (for new features in 21.3 series)
- [21.3.1 release notes](https://gitlab.freedesktop.org/mesa/mesa/-/blob/main/docs/relnotes/21.3.1.rst)

### CODE REVIEW

**Review by alalazo — APPROVED** [review]

**Comment by alalazo** [comment]
Waiting for package maintainers :heavy_check_mark: before merging

**Review by v-dobrev — APPROVED** [review]

**Pull request merged**
